code/game/objects/items/food/egg.dm

```diff
@@ -85,6 +85,24 @@
 	else
 		..()
 
+/obj/item/food/egg/afterattack_secondary(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+
+	if(istype(target,/obj/machinery/griddle))
```

Comment:
Also, early return this?

Reply:
Doesn't it return anyway since after the if() block is the end of the procedure?